p/api/greader.php

```diff
@@ -492,8 +492,8 @@ function entriesToArray($entries) {
 		}
 		$item = array(
 			'id' => 'tag:google.com,2005:reader/item/' . dec2hex($entry->id()),	//64-bit hexa http://code.google.com/p/google-reader-api/wiki/ItemId
-			'crawlTimeMsec' => substr($entry->id(), 0, -3),
-			'timestampUsec' => '' . $entry->id(),	//EasyRSS
+			'crawlTimeMsec' => '' . ($entry->date(true) * 1000),
+			'timestampUsec' => '' . ($entry->date(true) * 1000000),	//EasyRSS
```

Comment:
Since it's also required for Feeder (?) it might be a good idea to indicate that in the comment.

Reply:
I included Reeder in the comment. To be noted: 
- I tested with Reeder (macOS & iOS) with lasts versions, it works well.
- I didn't test with EasyRSS, as I don't have Android device. Perhaps it would be good to make a quick test with it, to be 100% sure it don't break anything.